Algebraic Topology/algebraic-topology.tex

```diff
@@ -431,17 +436,202 @@ \section{Functoriality of $\fund$}
+
+  \includegraphics[scale=0.2]{images/trefoil-fully-described.jpg}
+\end{example}
+\begin{thm}[Wirtinger]
+  For any knot $K$, $\fund(\R^3\minus K)$ is generated by loops
+  $\gamma_1,,,\gamma_m$ over crossings modulo $a = cbc^{-1}$.
+\end{thm}
+
+
+\section{CW complexes}
+
+CW complexes are like a space built by gluing together balls.
+\begin{defn}
+  A \de{CW complex} is a space $X$ which is a union of
+  $X_0 \subset X_1 \subset \subset \subset$ such that $X_n$ is
+  constructed from $X_{n-1}$ by taking
+  $X_{n-1} \dunion_{\alpha} D_{\alpha}^n$ modulo identifying (( --> he
+  might have meant $S^n$ here.  i think the point is that he meant
+  $\d D^n$)) $\phi_\alpha \colon S_\alpha^{n-1} \to X_{n-1}$.  $X_n$
+  is called $n$-Skeleton.  $X ~ \phi_\alpha$.
+
+  with topology s.t. $A$ is open iff $A \cap X_n$ is open for all $n$.
+\end{defn}
+\begin{example}
+  $S^n = D^{n-1}/\text{boundary}$ all goes to one point.  $X_0 =$
+  discrete set of points.  The $0$-skeleton is a point.  $X_1 =$
+  graph. (by connecting the points in $X_0$.)  $X_2 =$ take each cycle
+  (can go around multiple times) in $X_1$ and glue it to the boundary
+  of a disk (can go around multiple times).
+\end{example}
+\begin{example}
+  ((this example is incomplete and will be repeated next lecture))
+  $\R P^n =$ lines in $\R^{n+1}$.
+
+  $\R P^n = \{ \R \dot (1, a_1,,, a_{n-1})\} \isom \R^{n}$.
+  $\bunion \{ \R \dot (0, a_1,,, a_{n-1})\} \isom \R P^{n-1}$.
+\end{example}
+
+\mydate{d7}{6}{2}{2017}
+
+We wish to show how \(\R P^n\) is a \(CW\)-complex. First let the
+\(n-1\) skeleton be \(\R P^{n-1}\). We add the \(n\) cell \(D^n\) to
+structure under the gluing map \(S^{n} \to \R P^{n-1}\) where
+antipodal points are identified. By induction this gives us a
+\(CW\)-complex structure.
+
+\begin{example}[Something that does not admit a \(CW\) structure]
+  The topological space known as the \emph{Hawaiian earring} is
+  constructed as follows for each \(n \in \Z\) consider the circle of
+  radius \(1/n\) tangent to the \(y\)-axis. All of these circles
+  together with the subspace topology yield a topological space, call
+  it \(X\).
+
+  The only hope for putting a \(CW\) structure on this pace would be
+  to consider one \(0\)-cell along with a countable amount of
+  \(1\)-cells. However recall that the \(CW\)-structure gain their
+  topology through the quotient topology, that is, \(\mathcal{O}\) is
+  open in \(X\) if and only if \(q^{-1}(X)\) is open, where \(q\) is
+  the glueing map. However any open set centered at the origin
+  contains an infinite amount of balls, and thus no such
+  \(CW\)-structure may be imposed.
+\end{example}
+
+Next we show how to compute \(\pi_1\) of a few
+\(CW\)-structures. Using Morse Theory one may show that any compact
+manifold admits a \(CW\)-structure.
+
+To compute \(\pi_1\) of the torus one can first decompose it as a
+\(CW\)-complex as follows
+\begin{align*}
+  X_0 &= *\\
+  X_1 &= S^1 \vee S^1\\
+  X_2 &= D_2
```

Comment:
Maybe he meant D^2 ?  I still don't get CW-complexes.  For the torus, X_0 and X_1 make sense, but this X_2 does not make sense to me.  Do we eventually get the torus itself (S^1 \x S^1) for one of the X_n's ?